zcash_client_backend/src/fees.rs

```diff
@@ -19,13 +20,20 @@ pub mod zip317;
 /// A proposed change amount and output pool.
 #[derive(Clone, Debug, PartialEq, Eq)]
 pub enum ChangeValue {
-    Sapling(Amount),
+    Sapling {
+        value: Amount,
+        memo: Option<MemoBytes>,
+    },
 }
 
 impl ChangeValue {
+    pub fn sapling(value: Amount, memo: Option<MemoBytes>) -> Self {
+        Self::Sapling { value, memo }
+    }
+
     pub fn value(&self) -> Amount {
         match self {
-            ChangeValue::Sapling(value) => *value,
+            ChangeValue::Sapling { value, .. } => *value,
         }
     }
 }
```

Comment:
```suggestion
    pub fn memo(&self) -> MemoBytes {
        match self {
            ChangeValue::Sapling { memo, .. } =>
                memo.as_ref().map_or_else(MemoBytes::empty, |m| m.clone())
        }
    }
}
```
(or however rustfmt formats that). Alternatively keep the return value as `Option<MemoBytes>`.